packages/argo-admin/tsconfig.json

```diff
@@ -1,10 +1,14 @@
 {
   "extends": "../../config/typescript/tsconfig.admin.json",
   "compilerOptions": {
-    "rootDir": "src",
+    "rootDir": "../",
```

Comment:
It is necessary because of the new `ui-extensions` is outside of `./argo-admin/src` folder. We don't want to consume `ui-extensions` as a standard publish library. That's why we need to change the rootDir. Changing it won't affect build files outside of the package because we already define `"include": ["./src/**/*.ts", "../ui-extensions/**/*.ts"],`.